Makefile

```diff
@@ -3,9 +3,13 @@ default: build
 build: 
 				docker-compose build
 
-run: 
+run-dev: 
 				-docker-compose down
-				docker-compose up
+				docker-compose -f docker-compose.yml -f docker-compose.dev.yml up
+
+run-prod:
+				-docker-compose down
+				docker-compose -f docker-compose.yml -f docker-compose.prod.yml up
```

Comment:
can we have this run in the background by default

Reply:
-d

Reply:
Sounds good to me. 

Just prod runs? Or do you want dev runs as well to be background? I'm only adjusting the prod for now, since I think its nice to see the output when running locally, but I'm open to anything.